### ISSUE TITLE
Bench does not terminate if using a benchmark group with AxisScale::logarithmic

### ISSUE THREAD

**Issue by Kannen** [bheisler/criterion.rs#471]
This bench will block at the end, apparently when criterion build the report.

The command used to run the bench is `cargo bench` or `cargo criterion`. 
Criterion normally generates html report if axis scale is linear. 
Environment: GnuPot is not installed, the pc is a x86_64 linux.

```
fn bench_init_mut_lazy(c: &mut Criterion) {
    let mut gp = c.benchmark_group("A group");

    //BUG if not commented out:
    gp.plot_config(PlotConfiguration::default().summary_scale(AxisScale::Logarithmic));

    gp.bench_with_input(
        BenchmarkId::new("A", 1),
        &1,
        |b,v| b.iter_batched(
            || *v,
            |v| v*2,
            BatchSize::SmallInput
            )
        );
    gp.bench_with_input(
        BenchmarkId::new("A", 2),
        &2,
        |b,v| b.iter_batched(
            || *v,
            |v| v*2,
            BatchSize::SmallInput
            )
        );
    gp.finish()
}
```

**Comment by Kannen** [comment]
There is no bug if GnuPlot is installed.

**Comment by vadixidav** [comment]
Just encountered this same issue. :+1:

**Comment by YuhanLiin** [comment]
Happened to me as well.

**Comment by bheisler** [comment]
This seems to be an issue with the plotters backend. Would recommend disabling that if you encounter this. It seems to be happening with cargo-criterion as well, see https://github.com/bheisler/cargo-criterion/issues/33

@38 Do you know anything about this?

**Comment by lemmih** [comment]
Fixed in 0.3.5 by bumping the plotters dependency.
